docs/getting-started/install/azure.md

```diff
@@ -0,0 +1,17 @@
+# Microsoft Azure
+
+- Generate an SSH key pair by executing:
```

Comment:
This section should say "generate a key if you don't already have one" or something similar. It would be nice to link to the github docs on key generation as well :)